README.md

```diff
@@ -401,15 +401,12 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 
 ## Artificial Intelligence / Machine Learning / Big Data
 
-- [Data Science Weekly Newsletter](https://www.datascienceweekly.org/). A free weekly newsletter featuring curated news, articles and jobs related to Data Science.
-- [Data Eng Weekly](https://dataengweekly.com). Your weekly Data Engineering news (Formerly Hadoop Weekly). [Archive](https://dataengweekly.com/archive.html).
-- [Data Elixir](https://dataelixir.com/). A weekly newsletter of the best data science news and resources from around the web. [Archive](https://dataelixir.com/newsletters/).
```

Comment:
https://dataelixir.com/newsletter-archives/ it's alive, can we put it back?

Reply:
sure sure thanks !